Dockerfile.deploy

```diff
@@ -10,10 +10,21 @@ RUN useradd --no-create-home -s /bin/bash ${ALMIGHTY_USER_NAME}
 COPY bin/alm ${ALMIGHTY_INSTALL_PREFIX}/bin/alm
 COPY config.yaml ${ALMIGHTY_INSTALL_PREFIX}/etc/config.yaml
 
+# Install little pcp pmcd server for metrics collection
+# would prefer only pmcd, and not the /bin/pm*tools etc.
+COPY pcp.repo /etc/yum.repos.d/pcp.repo
+RUN yum install -y pcp && yum clean all
+COPY ./alm+pmcd.sh /alm+pmcd.sh
+RUN chmod a+x      /alm+pmcd.sh
```

Comment:
The chmod may be redundant; the file already has 755 perms.  I've had problems on some hand-driven docker invocations without the chmod, so added it here defensively.
As to the cost of pmcd, it takes some 10MB of memory and zero CPU when it is not actively polled. When polled at the sorts of rates used by pcp defaults (60-seconds, a few dozen metrics), it will use a tiny fraction of CPU.